futures-cpupool/src/lib.rs

```diff
@@ -179,22 +181,18 @@ impl CpuPool {
         Builder::new().create()
     }
 
-    fn execute_run(&self, run: Run) {
```

Comment:
Clarification: there are no functional changes to the behavior of this file, right? It's just refactoring?

Reply:
Ah the change here was that each instance of `Run` contained a `CpuPool` and it was switched to an `Arc<Inner>` because the destructor of `CpuPool` was significant for shutting it down. Other than that though no big changes!